src/components/about-us/Timeline.tsx

```diff
@@ -130,49 +138,122 @@ export const Timeline: React.FC = () => {
         >
           Our History
         </Heading>
+        <Flex
+          direction="row"
+          gap="7"
+          justify="center"
+          display={{ initial: "none", sm: "flex" }}
+        >
+          <Link href="#2024" size="4">
```

Comment:
Normally, I'd say extract the element and make a reusable one but these are so small that it would take more lines of code to do that than it takes to just repeat these links like that. Well done.